packages/frontend-shared/cypress/support/mock-graphql/mountFragment.ts

```diff
@@ -123,7 +123,7 @@ export const registerMountFn = ({ plugins }: MountFnOptions = {}) => {
                 source: print(source),
               }))
             },
-          }).end()
+          })?.end()
```

Comment:
@AtofStryker  why were the graphql change around the log instance needed?

Reply:
I believe because `Cypress.log` might return undefined and was failing the `check-ts` job. Doesn't seem related to the changes for `cy.origin`, but my guess is [this](https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cypress/log.ts#L634) is the reason for this, but this also seems to already be in the branch, so I am a bit unsure. @tgriesser are you able to shed some light here?

Reply:
Its probably picking up these type changes: https://github.com/cypress-io/cypress/blob/develop/packages/driver/types/cypress/log.d.ts#L4

Reply:
OK now that makes sense! Still seems strange to me a constructor type function would return `undefined`, but sounds like that's out of scope here 😅 .